games/2048.cpp/files/data_path.diff

```diff
@@ -0,0 +1,103 @@
+--- CMakeLists.txt.ori	2020-03-31 17:44:44.000000000 +0800
++++ CMakeLists.txt	2020-03-31 17:59:32.000000000 +0800
+@@ -31,4 +31,4 @@
+   RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin)
+ 
+ install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/data
+-        DESTINATION ${CMAKE_INSTALL_PREFIX})
++        DESTINATION ${CMAKE_INSTALL_PREFIX}/var/tmp/2048)
```

Comment:
I don't think I've seen any other ports use ${prefix}/var/tmp for anything. How about ${prefix}/var/${name} instead?

It looks like the game wants to save persistent data to this location. Since the game could be launched by any user, you'll have to have the port modify the permissions of the directory so that any user could write to it. And you'll probably use `destroot.keepdirs` to ensure the directory exists.

Does the game save its files to that directory such that they are writeable by any user? If not, might have to change the game code to do that.

Reply:
I have been able to change the path as well as its permissions with cmake.
Thanks for your advice.